### PR TITLE
Always notify user and keep going if some commits fail

### PR DESCRIPTION
**Pull request by ptare**
Let me know if you think this'll be too spammy.

Signed-off-by: Prasad Tare <prasad.tare@gmail.com>

### CODE REVIEW

File: core/scripts/Voting.js [hunk 1]
```diff
@@ -190,15 +192,22 @@ class VotingSystem {
...
       // Skip commits if a message already exists for this request.
       // This does not check the existence of an actual commit.
       if (await this.getMessage(request, roundId)) {
+        skipped.push(request);
         continue;
       }
 
-      commitments.push(await this.constructCommitment(request, roundId));
+      try {
+        commitments.push(await this.constructCommitment(request, roundId));
```
Comment (mrice32) [inline core/scripts/Voting.js]: One error case I think we should consider for a follow-up is when the commit or reveal txns fail.

File: core/scripts/Voting.js [hunk 2]
```diff
@@ -41,7 +41,9 @@ async function fetchCryptoComparePrice(request) {
 
   // Temporary price feed until we sort historical data.
   // CryptoCompare provides historical hourly prices for free. If we want minutes/seconds, we'll have to update later.
-  const url = `https://min-api.cryptocompare.com/data/histohour?fsym=${identifier.first}&tsym=${identifier.second}&toTs=${time}&limit=1&api_key=${CC_API_KEY}`;
+  const url = `https://min-api.cryptocompare.com/data/histohour?fsym=${identifier.first}&tsym=${
```
Comment (mrice32) [inline core/scripts/Voting.js]: Did something change here? Or was this just a formatting thing?

Comment (ptare) [reply core/scripts/Voting.js]: Just a formatting thing that prettier did. I've reverted the change.

File: core/scripts/Voting.js (hunk 1 above)
Comment (ptare) [reply core/scripts/Voting.js]: I agree, we should notify on that error case too. I'll add that in a follow up.

**Pull request merged**